app/src/main/java/amazingme/activities/app/ChildRegistrationActivity.java

```diff
@@ -1,31 +1,32 @@
 package amazingme.activities.app;
 
 import android.view.View;
-import android.widget.AdapterView;
```

Comment:
the binding for this page to the xml was already done. any reason this file was edited?